### PR TITLE
Fix incorrect truthy check failing on ID 0 coords

### PR DESCRIPTION
**Pull request by KrisDavie**

### CODE REVIEW

**Review by dweemx — APPROVED** [review]
This can probably be improved because logic behind is not easy to read imo. But for now is fine I think until we refactor `Compare.jsx`

File: src/components/pages/Compare.jsx [hunk 1]
```diff
@@ -402,7 +402,10 @@ class Compare extends Component {
                                             superposition={superposition}
                                             activeCoordinates={
                                                 configuration == 'multi'
-                                                    ? multiCoordinates[j]
+                                                    ? multiCoordinates[j] !=
+                                                          null &&
+                                                      multiCoordinates[j] !=
+                                                          undefined
                                                         ? multiCoordinates[j]
                                                         : -1
```
Comment (MaybeJustJames) [inline src/components/pages/Compare.jsx:410]: This is very very hard to parse. How about this:
```javascript
configuration === 'multi' ? R.defaultTo(-1, multiCoordinates[j]) : multiCoordinates[0]
```

**Review by MaybeJustJames — CHANGES_REQUESTED** [review]

**Review by MaybeJustJames — APPROVED** [review]
